src/modules/ekf2/ekf2_params.c

```diff
@@ -770,15 +789,30 @@ PARAM_DEFINE_FLOAT(EKF2_RNG_GATE, 5.0f);
 PARAM_DEFINE_FLOAT(EKF2_MIN_RNG, 0.1f);
 
 /**
- * Whether to set the external vision observation noise from the parameter or from vision message
+ * External vision (EV) noise mode
  *
- * If set to true the observation noise is set from the parameters directly, if set to false the measurement noise is taken from the vision message and the parameter are used as a lower bound.
+ * If set to 0 (default) the measurement noise is taken from the vision message and the EV noise parameters are used as a lower bound.
+ * If set to 1 the observation noise is set from the parameters directly,
  *
- * @boolean
+ * @value 0 EV reported variance (parameter lower bound)
+ * @value 1 EV noise parameters
  * @group EKF2
  */
 PARAM_DEFINE_INT32(EKF2_EV_NOISE_MD, 0);
 
+/**
+ * External vision (EV) minimum quality (optional)
+ *
+ * External vision will only be started and fused if the quality metric is above this threshold.
+ * The quality metric is a completely optional field provided by some VIO systems.
+ *
+ * @group EKF2
+ * @min 0
+ * @max 100
```

Comment:
The uorb message allocates a `uint8` for the quality field, is it usually a value between 0 and 255 or 0 and 100%?

Reply:
It comes from mavlink and intended to be a percentage. http://mavlink.io/en/messages/common.html#ODOMETRY

At the moment it's intended as a mechanism to prevent the bad stuff from being used, but maybe in the future we'll want to do more.